geode-core/src/main/java/org/apache/geode/management/internal/cli/CliUtil.java

```diff
@@ -133,13 +142,12 @@ private static boolean canLoadClass(String className) {
     allClusterMembers.add(cache.getDistributedSystem().getDistributedMember());
 
     for (DistributedMember member : allClusterMembers) {
-      for (String regionAssociatedMemberName : regionAssociatedMemberNames) {
-        String name = MBeanJMXAdapter.getMemberNameOrUniqueId(member);
-        if (name.equals(regionAssociatedMemberName)) {
-          matchedMembers.add(member);
-          if (!returnAll) {
-            return matchedMembers;
-          }
+      List<String> regionAssociatedMemberNamesSet = Arrays.asList(regionAssociatedMemberNames);
+      String name = MBeanJMXAdapter.getMemberNameOrUniqueId(member);
+      if (regionAssociatedMemberNamesSet.contains(name)) {
+        matchedMembers.add(member);
+        if (!returnAll) {
+          return matchedMembers;
```

Comment:
We didn't end up needing the changes made to this file for our work, but the above refactor is an improvement and we should keep it.